javascript/javascript2/week1/shortest_word.js

```diff
@@ -0,0 +1,16 @@
+
+
+//---------------------find_the_shortest_word-----------------------------//
+
+function find_shortest_word(danishWords) {
+    danish_letter = danishWords;
+    words = danishWords.replace(",", " ").split(" ");
+    words.forEach(function (word) {
+        if (word.length < danish_letter.length) {
+            danish_letter = word
+        };
+    });
+    return danish_letter
+};
+
+console.log(find_shortest_word("bil plante kaffe bog ø planetarium"));
```

Comment:
Good work! what if the text contains a number? does it still return shortest word or it returns number?